Annotations/Classes/Model/ArrowModel.swift

```diff
@@ -28,22 +29,29 @@ public struct ArrowModel: Model {
   func copyMoving(arrowPoint: ArrowPoint, delta: PointModel) -> ArrowModel {
     switch arrowPoint {
     case .origin:
-      return ArrowModel(origin: origin.copyMoving(delta: delta), to: to, color: color)
+      return ArrowModel(index: index,
+                        origin: origin.copyMoving(delta: delta),
+                        to: to, color: color)
```

Comment:
@dworak each function parameter should be started from a new line for better readability